src/data/auth/projectServices.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  ProjectsData,
+  NewProject,
+  DeleteProject,
+  Project,
+} from '../../types/project';
+import { http } from '../http/http';
+
+export const getAllProjects = async (): Promise<ProjectsData> => {
+  const { data } = await http.get<ProjectsData>('/projects');
```

Comment:
'/projects' path is reusable part for all projects routes ⇾ it could be moved to `PROJECTS_BASE_PATH` constant

Reply:
I have http constant with base url for all routes